Assets/Scripts/AttachThePlayer.cs

```diff
@@ -0,0 +1,30 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AttachThePlayer : MonoBehaviour {
```

Comment:
[ADVICE]
A class name should not be a combine of "verb. + noun.".
Maybe just Attach under a namespace of Enemy is better.